scrapybot/scrapybot/spiders/bolts.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+from .utils import strip_tags
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+
+class BoltsSpider(CrawlSpider):
+    name = "bolts"
+    allowed_domains = ["github.com"]
+    start_urls = ["https://github.com/lightning/bolts/blob/master/00-introduction.md"]
+
+    rules = (
+        Rule(
+            LinkExtractor(restrict_xpaths="//ol/li/a"),
+            callback="parse_item",
+        ),
+    )
+
+    def parse_item(self, response):
+        item = {}
+
+        body_to_be_parsed = response.xpath("//article").get()
+        item["id"] = "bolts-" + str(uuid.uuid4())
+        item["title"] = response.xpath('//h1[@dir="auto"]/text()').get()
+        item["body"] = strip_tags(body_to_be_parsed)
```

Comment:
so for the asciidoc and markdown files, it seems like you stripped the headers/tags and uploaded the raw text or did you capture the headers in another way?

Reply:
in the [new setup](https://github.com/bitcoinsearch/scraper/pull/30) i capture the immediate html tags as well as the raw text